tests/cases/compiler/instanceOfAssignability.ts

```diff
@@ -0,0 +1,54 @@
+interface Base {
+	foo: string|number;

```

Comment:
Can you please change your editor's tab settings? It makes for a more consistent diff.